schema.sql

```diff
@@ -12,3 +12,24 @@ CREATE TABLE animals (
 /* Add a new column to the database */
 ALTER TABLE animals ADD COLUMN species VARCHAR(250);
 
+/* ADD new tables */
+
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(255),
+  age INT
+);
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) 
+);
+
+ALTER TABLE animals ALTER COLUMN id SET NOT NULL;
+ALTER TABLE animals ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY;
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id INT;
```

Comment:
- [ ] This query should reference `species` table

## Sample code:
``ALTER TABLE animals ADD species_id int REFERENCES species(id);``